docs/modules/opa/examples/getting_started/getting_started.sh.j2

```diff
@@ -1,6 +1,16 @@
 #! /usr/bin/env bash
 set -euo pipefail
 
+# DO NOT EDIT THE SCRIPT
+# Instead, update the j2 template, and regenerate it for dev:
+# cat <<EOF | jinja2 --format yaml getting_started.sh.j2 -o getting_started.sh
```

Comment:
Can we put this in Make instead of requiring people to re-run jinja manually for specific files?

Reply:
Sure. I didn't even know it was there so I've added this warning to all.
I will keep doing so for consistency, but will make an issue to fix this up after the release.

Reply:
https://github.com/stackabletech/issues/issues/611